scripts/shipTemplates_Corvette.lua

```diff
@@ -232,68 +232,68 @@ template:setBeam(5, 30, 300, 4000.0, 1.5, 20)
 --[[----------------------Freighters----------------------]]
 
 for cnt=1,5 do
-    template = ShipTemplate():setName("Personnel Freighter " .. cnt):setClass("Corvette", "Freighter"):setModel("transport_1_" .. cnt)
+    template = ShipTemplate():setName("Personnel Freighter " .. cnt):setLocaleName(_("Personnel Freighter ") .. cnt):setClass(_("Corvette"), _("Freighter")):setModel("transport_1_" .. cnt)
```

Comment:
Instead of `_("Personnel Freighter ") .. cnt`
It would be better to use `_("Personnel Freighter %d").format(cnt)`, as that gives the translator the decision on where to put the number.

Reply:
Oups, my mistake.

Reply:
My bad !

I fixed it ;)